src/components/NcAppSidebar/NcAppSidebarTabs.vue

```diff
@@ -90,6 +90,8 @@ export default {
 			unregisterTab: this.unregisterTab,
 			// Getter as an alternative to Vue 2.7 computed(() => this.activeTab)
 			getActiveTab: () => this.activeTab,
+			// Used to check whether the tab header is shown so the tabs can reference the tab header for `aria-labelledby` or not
+			tablistShown: this.hasMultipleTabs,
```

Comment:
This provide is not reactive. We need to provide a getter or a `computed`.

Reply:
Is `hasMultipleTabs` not a computed?

Reply:
Or do you mean it is required to be like this:
```
tablistShown: () => this.hasMultipleTabs
```

(`hasMultipleTabs` is a computed property)

Reply:
> Is `hasMultipleTabs` not a computed?

This is a computed value, but not a computed reference object.

From JS side, it is just a boolean value `true/false`, so "provide" provides the value "true" by key "tablistShown". We need to provide a reference to a reactive mutable object or a computed ref.

> `tablistShown: () => this.hasMultipleTabs`

This will work in any Vue, but must be used as a function

`getTablistShown: () => this.hasMultipleTabs`

Or it can be computed ref:
`tablistShown: computed(() => this.hasMultipleTabs)`, but Raimund had a problem with this solution earlier (though, it should work in Vue 2.7+)

Reply:
Ah I see Option API computed != reactive computes from Composition API :)

>  tablistShown: computed(() => this.hasMultipleTabs), but Raimund had a problem with this solution earlier (though, it should work in Vue 2.7+)

I like this one more, but looks also a bit like mixing Options API and Composition API.

Reply:
> I like this one more, but looks also a bit like mixing Options API and Composition API.

Though Vue 3 reactivity with `reactive`, `ref`, `computed` was created for Composition API, it is not Composition API itself, not mixing. There is still no `setup` in the component, so no component is described with Composition API. This is just a reactivity function and a recommended way to provide reactive values in Vue 3 (if not count new toRef from 3.3 update).

https://vuejs.org/guide/components/provide-inject.html#working-with-reactivity

In Vue 3 there is even unwrapping for injected refs/computed in Options API.